docs/extend/manifest.md

```diff
@@ -50,11 +50,11 @@ Manifest provides the base accessible fields for working with V0 plugin format
       
     - **`types`** *string array*
 
-      types indicate what interface(s) the plugin currently implements.
+          types indicate what interface(s) the plugin currently implements.
```

Comment:
This document is now published from https://github.com/docker/docker.github.io, so this change should be reverted, and reopened as a PR in that repository